server.js

```diff
@@ -10,8 +10,7 @@ const app = express(),
 	config = require('dotenv').config()
 
 if (config.error) {
-	console.warn('[ERROR]: cannot parse .env file')
-	process.exit(-1)
+	console.warn('[WARNING]: cannot parse .env file, falling back to systems ones')
```

Comment:
why? we can't run without the GitHub env variables, there are none to fallback to

Reply:
You try to parse a .env-file, which does not exist when you use Docker. Furthermore people may want to use the system-wide environment variables instead a .env file.

Reply:
Oh yeah forgot you don't have .env with docker

Reply:
.env is just a feature which reads environment variables out of a file (.env for example) and 'injects' them into your application, so that you can read them like normal system-wide variables. I would **never** force .env-files. It should **always** be an optional feature you offer.